packages/react/src/components/AspectRatio/AspectRatio.tsx

```diff
@@ -7,22 +7,58 @@
 
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { PropsWithChildren, ReactHTML } from 'react';
 import { usePrefix } from '../../internal/usePrefix';
 
+interface AspectRatioProps {
+  /**
+   * @description Provide a custom component or string to be rendered as
```

Comment:
The `@description` JSDoc annotation is [not supported](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#unsupported-tags)

```suggestion
   * Provide a custom component or string to be rendered as
```

Additional usages of `@description` in this PR will need updated as well